tmt/steps/prepare/feature/fips-disable.yaml

```diff
@@ -0,0 +1,19 @@
+- hosts: all
+  user: root
+  tasks:
+    # RHEL 7 or CentOS 7
+    - when: ansible_distribution_major_version == "7"
+      block:
+        - name: Enable fips
+          command: echo UNSUPPORTED
```

Comment:
Shouldn't we fail instead? 
Btw there is a way - https://github.com/beakerlib/crypto/blob/master/fips/lib.sh#L143 however not sure if it is worth implementing it.

Reply:
> Shouldn't we fail instead?

Yes. We should fail it if it is unsupported by tmt.

> Btw there is a way - https://github.com/beakerlib/crypto/blob/master/fips/lib.sh#L143 however not sure if it is worth implementing it.

Looks the implementation is complicated as it should take care of `Intel CPU`.  @psss and @happz, what's your idea?

Reply:
Good question. Should it fail on an unsupported distro? Or turn into a no-op? What should the "enable EPEL" playbook do when executed on Ubuntu guest? We don't fail there either... I have no clear idea, TBH, maybe there should be a block for unsupported combinations, raising an error, or quitting the playbook with an error, or just a warning... I can imagine all of these.

> > Btw there is a way - https://github.com/beakerlib/crypto/blob/master/fips/lib.sh#L143 however not sure if it is worth implementing it.
> 
> Looks the implementation is complicated as it should take care of `Intel CPU`. @psss and @happz, what's your idea?

My idea was to keep FIPS for later :) Especially because I know little about FIPS, and I prefer a gradual approach. That said, I think we should support it, as pointed out, there is a fairly simple script, playbook can run commands like `grep`, therefore checking CPU flags is doable. I don't think we need to support it *now*, this can be done in incremental steps - add a plugin, add CRB "feature", add FIPS "feature", add support for more complicated "FIPS on RHEL7" feature, add EPEL "feature", add X "feature, and so on. Since the `prepare/feature` plugin supports "plugins" in this way, we can add them independently.

Reply:
Thanks for your suggestion, @happz. I'm with you that we don't support FIPS in this PR. Let me remove the related code and just support both `EPEL` and `CRB` in this PR.